go/obscuronode/enclave/storage.go

```diff
@@ -149,7 +148,11 @@ func (s *storageImpl) StoreBlock(b *types.Block) {
 	} else {
 		bAndHeight, f := s.db.FetchBlockAndHeight(b.ParentHash())
```

Comment:
we shouldn't be needing these blockAndHeight structures now.
Do you want to remove them as part of this pr?
Or shall we put a todo for a further cleanup pr?